msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,626 @@
+      }
+    }
+    
+    // The output will be persisted in .env.{envName}. Visit https://aka.ms/teamsfx-actions/arm-deploy for more details.
+    output TAB_AZURE_APP_SERVICE_RESOURCE_ID string = tabModule.outputs.TAB_AZURE_APP_SERVICE_RESOURCE_ID // used in deploy stage
+    output TAB_DOMAIN string = tabModule.outputs.TAB_DOMAIN
+    output TAB_ENDPOINT string = tabModule.outputs.TAB_ENDPOINT
+    
+    output ADDIN_AZURE_STORAGE_RESOURCE_ID string = addinModule.outputs.ADDIN_AZURE_STORAGE_RESOURCE_ID // used in deploy stage
+    output ADDIN_DOMAIN string = addinModule.outputs.ADDIN_DOMAIN
+    output ADDIN_ENDPOINT string = addinModule.outputs.ADDIN_ENDPOINT
+    ```
+
+1. In Visual Studio Code open the Teams Toolkit and in the **ACCOUNTS** section be sure you're signed into your *Azure* account (in addition to being signed into your Microsoft 365 account). For more information about signing in, open [Exercise - Create Azure resources to host a Teams tab app](/training/modules/teams-toolkit-vsc-deploy-apps/03-create-azure-resources-exercise) and scroll to the **Sign in to Azure in Teams Toolkit** section.
+1. In the **LIFECYCLE** section of Teams Toolkit, select **Provision**. It can take several minutes. You may be prompted to select one of your Azure resource groups.
+1. When provisioning completes, select **Deploy** to deploy your app code to Azure.
+
+### Run the tab capability from the remote deployment
+
+1. Select **View** | **Run** in Visual Studio Code and in the drop down menu, select one of the following choices:
+
+    - **Launch Remote in Teams (Edge)**
+    - **Launch Remote in Teams (Chrome)** 
+
+1. Press F5 to preview your Teams tab capability.
+
+### Run the add-in capability from the remote deployment
+
+1. Copy the production URL from the ADDIN_ENDPOINT in env/.env.dev file.
+1. Edit \add-in\webpack.config.js file and change `urlProd` constant value to the value you just copied. Be sure to add a '/' at the end of the URL.
+1. In the Visual Studio Code **TERMINAL**, navigate to the root of the project, and then run `npm run build:add-in`.
+1. Copy the file \add-in\dist\manifest.dev.json to the \appPackage folder.
+1. Rename the copy in the \appPackage folder to "manifest.addinPreview.json".
+1. In the **TERMINAL**, run `npx office-addin-dev-settings sideload .\appPackage\manifest.addinPreview.json`. This process can take a couple of minutes. Eventually, Outlook desktop opens. (If you're prompted to install `office-addin-dev-settings`, respond "yes".)
+1. Open the **Inbox** of *your Microsoft 365 account identity* and open any message. A **Contoso Add-in** tab with two buttons appears on the **Home** ribbon (or the **Message** ribbon, if you open the message in its own window).
+1. Select the **Show Taskpane** button and a task pane opens. Select the **Perform an action button** and a small notification appears near the top of the message.
+
+## What’s next
+
+There are other commonly suggested next steps, for example:
```

Comment:
Add the below options as bulleted list.

Reply:
I don't understand. The below options already are a bulleted list.